raw-svo.valid.main.lua

```diff
@@ -4544,12 +4544,23 @@ function valid.devastate_arms_cripple()
 end
 
 -- happens on wrist fracture levels 4,5
+-- edit: This ability can also mutilate a mangled limb.
 function valid.devastate_arms_mangle()
   checkaction(dict.wristfractures.gone, true)
   lifevision.add(actions.wristfractures_gone.p)
 
-  valid.simplemangledrightarm()
-  valid.simplemangledleftarm()
+  if affs.mangledrightarm
+  then
+    removeaff("mangledrightarm")
+	valid.simplemutilatedrightarm()
+  else valid.simplemangledrightarm()
+  end
+  if affs.mangledleftarm
+  then
+    removeaff("mangledleftarm")
+	valid.simplemutilatedleftarm()
```

Comment:
We use 2 spaces instead of tabs. See https://github.com/svof/svof/wiki/Code-style